styles.css

```diff
@@ -9,37 +9,42 @@ table {
     We've coupled this with a width of 100%, meaning that the table will fill any container it is put in, and be nicely
     responsive (although it would still need some more work to get it looking good on narrow screen widths).
      */
-    table-layout: fixed;
-    width: 100%;
-    /*
+  table-layout: fixed;
+  width: 100%;
+  /*
     A border-collapse value of collapse is standard best practice for any table styling effort.
     By default, when you set borders on table elements, they will look pretty ugly.
      */
-    border-collapse: collapse;
-    border: 3px solid purple;
+  border-collapse: collapse;
+  border: 3px solid lightblue;
+}
+
+tr:nth-child(1) {
```

Comment:
You are duplicating `tr:nth-child(1)` block - it is already in the file in line 48.